evaluator/evaluator.py

```diff
@@ -60,6 +61,16 @@ def _prepare_error_key_map(self):
         cur.close()
         self.conn.commit()
         return error_key_map
+
+    def _prepare_impact_id_map(self):
+        impact_id_map = {}
+        cur = self.conn.cursor()
+        cur.execute("select name, id from cve_impact")
+        for impact_name, impact_id in cur.fetchall():
+            impact_id_map[impact_name] = impact_id
+        cur.close()
+        self.conn.commit()
```

Comment:
Is there need to this commit? I mean we're just selecting data and not inserting anything.

Reply:
@tkasparek thank you, good point. looks like a copypaste from `_prepare_error_key_map`.
@eherget could, you, please, remove this `commit()` from both methods?

Reply:
unnecessary commits removed.